rome_app/urls_app.py

```diff
@@ -28,6 +29,8 @@
     url(r'^essays/(?P<essay_slug>\w+)/$', views.essay_detail, name='specific_essay'),
     url(r'^people/$', views.biography_list, name='people'),
     url(r'^people/(?P<trp_id>\d+)/$', views.biography_detail, name='person_detail'),
+    url(r'^shop_list/$', views.shop_list, name='shops'),
```

Comment:
This name of "shops" is already used above. Need to use a unique name. Be sure to update your test below with the new url name.